packages/composable-controller/src/ComposableController.test.ts

```diff
@@ -10,7 +10,8 @@ import {
 import type { Patch } from 'immer';
 import * as sinon from 'sinon';
 
-import type { ComposableControllerStateChangeEvent } from './ComposableController';
+import { JsonRpcEngine } from '../../json-rpc-engine/src/JsonRpcEngine';
```

Comment:
Nit: We have Jest and TypeScript set up so that if you use `@metamask/*` it will look for an appropriate package in `packages/`. What are your thoughts on:

``` suggestion
import { JsonRpcEngine } from '@metamask/json-rpc-engine';